js/build.js

```diff
@@ -1,45 +1,55 @@
-(function(){
-  window.ui = window.ui || {}
+(function() {
+  window.ui = window.ui || {};
   ui.flipletCharts = ui.flipletCharts || {};
 
   Fliplet.Chart = Fliplet.Widget.Namespace('chart');
 
   function init() {
-    Fliplet.Widget.instance('chart-pie-1-1-0', function (data) {
+    Fliplet.Widget.instance('chart-pie-1-1-0', function(data) {
       var chartId = data.id;
-      var chartUuid = data.uuid;
       var $container = $(this);
+      var themeInstance = Fliplet.Themes.Current.getInstance();
+      var themeValues = themeInstance.data.values ? Object.assign({}, themeInstance.data.values) : {};
```

Comment:
What are the possible values of `themeInstances.data.values` that you are managing here?

If `themeInstance.data.values` is `undefined`, `Object.assign({}, undefined)` produces `{}`. Why are the `?` and `:` cases necessary?

![image](https://user-images.githubusercontent.com/290733/131566896-37ad761f-dc8c-4881-938e-2fb4bd7a98f3.png)

Reply:
Yeah, you're right. I'll fix this